assets/textures/gui/bevel/Eruption.svg.import

```diff
@@ -2,24 +2,24 @@
 
 importer="texture"
 type="CompressedTexture2D"
-uid="uid://db157djiuw5ek"
-path="res://.godot/imported/Eruption.png-695a8e31f95ce98bf6964cabe9f4376f.ctex"
+uid="uid://c8mvyi3d47yvq"
+path="res://.godot/imported/Eruption.svg-c7b14ec729167b5aac84493ca88dce1d.ctex"
 metadata={
 "vram_texture": false
 }
 
 [deps]
 
-source_file="res://assets/textures/gui/bevel/Eruption.png"
-dest_files=["res://.godot/imported/Eruption.png-695a8e31f95ce98bf6964cabe9f4376f.ctex"]
+source_file="res://assets/textures/gui/bevel/Eruption.svg"
+dest_files=["res://.godot/imported/Eruption.svg-c7b14ec729167b5aac84493ca88dce1d.ctex"]
 
 [params]
 
 compress/mode=0
 compress/high_quality=false
 compress/lossy_quality=0.7
 compress/hdr_compression=1
-compress/normal_map=0
+compress/normal_map=2
```

Comment:
I just checked and only a few of the game textures have `compress/normal_map=2` with none of the GUI files having it. So this is definitely a setting that isn't used anywhere else in the GUI part of the game.